src/components/ScreenWrapper/index.js

```diff
@@ -85,6 +85,8 @@ class ScreenWrapper extends React.Component {
     }
 
     render() {
+        const maxHeight = this.props.shouldEnableMaxHeight ? this.props.windowHeight : null;
```

Comment:
NAB: should we use `undefined` instead of `null` here?

Reply:
True, is better to use `undefined` to remove styles, thanks